cms/models.py

```diff
@@ -41,7 +41,15 @@ def get_context(self, request):
         username = get_social_username(request.user)
         context = super(HomePage, self).get_context(request)
 
-        context["programs"] = programs
+        def get_program_page(program):
+            """Return a None if ProgramPage does not exist, to avoid template errors"""
+            try:
+                return program.programpage
+            except ProgramPage.DoesNotExist:
+                return
```

Comment:
This line is covered by tests: https://codecov.io/gh/mitodl/micromasters/src/d85d499840a690bc523c55c3afe6b589dd564c63/cms/models.py#L49

But it looks like there's no specific test for the presence of `ProgramPage` so I'll add one